src/provider/provider_os_memory.c

```diff
@@ -422,12 +408,10 @@ static umf_result_t os_alloc(void *provider, size_t size, size_t alignment,
     }
 
     if (alignment && (alignment % page_size) && (page_size % alignment)) {
-        if (os_provider->traces) {
-            fprintf(stderr,
-                    "wrong alignment: %zu (not a multiple or a divider of the "
-                    "minimum page size (%zu))\n",
-                    alignment, page_size);
-        }
+        LOG_DEBUG("wrong alignment: %zu (not a multiple or a divider of the "
```

Comment:
LOG_ERROR? we are returning an error here after all

Reply:
Done.